pages/client-libraries/java.mdx

```diff
@@ -4,105 +4,687 @@ description: Peruse our guide on implementing Java in Memgraph. Comprehensive do
+}
+```
+You can access the `Relationship` properties in the same way as you access the `Node` properties. Id values from `startNodeId()` and `endNodeId()` are the internal IDs of the start and end node of the relationship, do not used them in your application logic.
+
+
+### Processing Path results 
+
+You can receive `Path` from the database, using the following construct:
+
+```java
+public List<Record> readPath() {
+    String query = "MATCH path=(d:Developer)-[r:LOVES]->(t:Technology) RETURN path";
+    try (Session session = driver.session(SessionConfig.forDatabase("memgraph"))) {
+        List<Record> result = session.readTransaction(tx -> {
+            Result queryResult = tx.run(query);
+            List<Record> records = queryResult.list();
+            return records;
+        });
+        return result;
+    }
+}
+
+for (Record path : path_records) {
+    Path p = path.get("path").asPath();
+    System.out.println(p);
+    System.out.println(p.start());
+    System.out.println(p.end());
+    System.out.println(p.length());
+    System.out.println(p.nodes());
+    System.out.println(p.relationships());
+}
+```
+
+
+Path will contain [Nodes](#process-the-node-result) and [Relationships[#process-the-relationship-result], that can be accessed in the same way as in the previous examples, by casting them to the relevant type.
+
+#### Types mapping Cypher and Java 
+
+ Here is the full table of the mapping between Memgraph Cypher types and the types used in the Java driver:
+
+|  Cypher Type | Driver Type        |
+|-------------:|:-------------------|
+|          Null| null               |
+|       String | String             |
+|      Boolean | boolean            |
+|      Integer | long               |
+|        Float | double             |
+|         List | List(Object)       |
+|          Map | Map < String, Object >  |
+|         Node | Node               |
+| Relationship | Relationship       |
+|         Path | Path               |
+|     Duration | IsoDuration        |
+|         Date | LocalDate          |
+|    LocalTime | LocalTime          |
+|LocalDateTime | LocalDateTime      |
+
+
+Keep in mind that Memgraph does not support Timezones, ByteArray and Point at the moment. 
+
+### Transaction management
+
+Transaction is a unit of work that is executed on the database, it could be some basic read, write or complex set of steps in form of series of queries. There can be multiple ways to mange transaction, but usually, they are managed automatically by the driver or manually by the explicit code steps. Transaction management defines how to handle the transaction, when to commit, rollback, or terminate it.
+
+#### Simple transaction management  
+
+Simple transaction managment provide a standard blocking style API for Cypher execution. Simple transaction managment provides the easiest programming style to work with since API calls are executed in a strictly sequential fashion.
```

Comment:
```suggestion
Simple transaction management provides a standard blocking style API for Cypher execution. It also provides the most straightforward programming style to work with since API calls are executed strictly sequentially.
```